recordedit/form.controller.js

```diff
@@ -104,6 +104,29 @@
             $log.info(response);
         }
 
+
+        /*
```

Comment:
@karlcz  For `text` and `longtext` fields should we send null in case when the user empties the input ?